website/docs/essays/lifecycle.mdx

```diff
@@ -77,14 +78,14 @@ The process of verification involves checking the following items:
     1. deps should not be duplicated
 7. outputs_data_verifier
     1. number of ‘output data' fields must equal number of outputs
-8. since     
-    1. ‘since’ value must follow the rules described in https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0017-tx-valid-since/0017-tx-valid-since.md
+8. since
+    1. `since` value must follow the rules described in [tx-valid-since](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0017-tx-valid-since/0017-tx-valid-since.md)
```

Comment:
```suggestion
    1. `since` value must follow the rules described in [RFC0017](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0017-tx-valid-since/0017-tx-valid-since.md)
```